docs/guides/Uniq Factory Management/factory-purchase-options-examples.md

```diff
@@ -0,0 +1,330 @@
+---
+title: 'Factory Purchase Options Examples'
+deploy: ['staging', 'mainnet']
+order: 2
+---
+
+
+# Factory Purchase Options Examples
+
+Here we provide some example `cleos` commands to set purchase options and to purchase using created options. JSON data from provided `cleos` commands can be copied and utilized as a payload for the transaction for your API library of choice
+
+-   [setprchsreq.a - set purchase requirement](../../contracts/NFT%20Contract/NFT%20Actions/setprchsreq.a.md)
+-   [purchase.a - purchase a token](../../contracts/NFT%20Contract/NFT%20Actions/purchase.a.md)
+
+::: info
+Note that throughout this page the factory IDs, token IDs, user group IDs and account names used are not real and you must replace them with actual values you are interested in
+:::
+
+## Simple UOS/USD pricing
+
+This example utilizes the `price` field to set the price to 50 UOS
+
+::: details setprchsreq.a
+```sh
+cleos push action eosio.nft.ft setprchsreq.a '[
+  {
+    "token_factory_id": 100,
+    "index": 0,
+    "price": "50.00000000 UOS",
+    "purchase_limit": null,
+    "promoter_basis_point": 100,
+    "purchase_option_with_uniqs": null,
+    "sale_shares": [],
+    "maximum_uos_payment": null,
+    "group_restriction": null,
+    "purchase_window_start": null,
+    "purchase_window_end": null,
+    "memo": ""
+  }
+]' -p factory.manager
+```
+:::
+
+To be able to purchase from such factory you utilize `purchase.a` action
+
+::: details purchase.a
+```sh
+cleos push action eosio.nft.ft purchase.a '[
+  {
+    "token_factory_id": 100,
+    "index": 0,
+    "max_price": "100.00000000 UOS",
+    "buyer": "alice",
+    "receiver": "alice",
+    "promoter_id": null,
+    "user_uniqs": null,
+    "memo": ""
+  }
+]' -p alice
+```
+:::
+
+This example utilizes the `price` field to set the price to 5 USD
```

Comment:
Can we add "purchase" sample for this as well? 
This "USD" price was one of the confusions we had with the Marketing team when they were trying to write the article and it was not clear if content creator set purchase option in usd, how user can purchase it [uos or usd]?